gapis/api/vulkan/find_issues.go

```diff
@@ -142,7 +115,7 @@ func (t *findIssues) Transform(ctx context.Context, id api.CmdID, cmd api.Cmd, o
 		var debugReportExtNameData api.AllocResult
 		hasDebugReport := false
 		for _, e := range exts {
-			if e.String() == debugReportExtension {
+			if debugReportExtension == strings.TrimRight(string(memory.CharToBytes(e.StringSlice(ctx, s).MustRead(ctx, cmd, s, nil))), "\x00") {
```

Comment:
if there is no better way to access the string, maybe add a small comment to clarify why this has to be so convoluted?

Reply:
Done.